cpp/include/kvikio/remote_handle.hpp

```diff
@@ -92,6 +93,7 @@ class S3Endpoint : public RemoteEndpoint {
   std::string _url;
   std::string _aws_sigv4;
   std::string _aws_userpwd;
+  struct curl_slist* _curl_header_list;
```

Comment:
This member needs initialization. Otherwise, if `aws_session_token` is not given, the statement `if (_curl_header_list)` is UB.
```suggestion
  curl_slist* _curl_header_list{};
```

Reply:
Done